test/conditionSpec.js

```diff
@@ -98,6 +98,22 @@ describe('Conditional Transfers', () => {
       assert.equal((yield this.plugin.getBalance()), '5', 'balance should increase by amount')
     })
 
+    it('should fulfill a transfer even if inital RPC failed', function * () {
+      nock('https://example.com')
+        .post('/rpc?method=send_transfer&prefix=peer.NavKx.usd.', [this.transfer])
+        .reply(500)
+
+      const fulfilled = new Promise((resolve) => this.plugin.on('outgoing_fulfill', resolve))
+
+      yield this.plugin.sendTransfer(this.transfer)
+        .catch((e) => assert.match(e.message, /Unexpected status code 500/))
```

Comment:
This is dangerous, because right now senders assume that if the `sendTransfer` method rejects, the payment was not accepted. If you want to make the plugin wait until the transfer times out before resolving, that would be fine. But the way this works right now, the `ilp-core` client and `ilp` sender will think the transfer did not go through.